### DISCUSSION TITLE
Address와 User의 관계가 필요해보입니다.

### DISCUSSION THREAD

**Discussion by wenodev** [CodeSoom/DDD-Kurly-Clone-Order#15] [Ideas]
- Address는 User와 관계가 필요하고 DeliveryInfo와 Address(destinationAddrss)의 관계는 1:1이 적합해보입니다.

![image](https://user-images.githubusercontent.com/35131529/139004360-3182b1f8-9a1c-44d8-b3b6-e035163492dc.png)

**Comment by joowankim** [comment]
Address가 주문서에 적혀지는 주소라고 생각하고 관계를 만들었었는데 다시 생각해보니 1:1이 맞을 것 같네요.
그런데 마찬가지로 Address는 주문서에  적히는, 주문서에 종속될 VO이기 때문에 주문서 작성 당시에 주문자가 입력한 배송지 정보와 회원과의 관계는 필요없을 것 같습니다.
혹시, Address와 User와의 관계가 필요하다고 생각하시는 이유를 알 수 있을까요?
